src/test/java/org/apache/ibatis/submitted/cglib_lazy_error/CglibNPETest.java

```diff
@@ -104,14 +104,13 @@ void testInsertBetweenTwoSelects() {
     try (SqlSession sqlSession = sqlSessionFactory.openSession()){
       PersonMapper personMapper = sqlSession.getMapper(PersonMapper.class);
       Person selected1 = personMapper.selectById(1);
-      Person selected2 = personMapper.selectById(2);
-      Person selected3 = personMapper.selectById(3);
       selected1.setId(4L);
       int rows = personMapper.insertPerson(selected1);
       Assertions.assertEquals(1, rows);
+
       selected1 = personMapper.selectById(1);
-      selected2 = personMapper.selectById(2);
-      selected3 = personMapper.selectById(3);
+      Person selected2 = personMapper.selectById(2);
+      Person selected3 = personMapper.selectById(3);
       Person selected4 = personMapper.selectById(4);
       Assertions.assertEquals(1, selected1.getId().longValue());
       Assertions.assertEquals(2, selected2.getId().longValue());
```

Comment:
Hi @yangfancoming @kazuki43zoo ,
I'm sorry for a belated review, but I think we should revert this one as well.
Although it looks redundant, it could be a necessary procedure to reproduce the target problem.

Reply:
Revert via #1605.